driver-core/src/test/java/com/datastax/driver/core/UserTypesTest.java

```diff
@@ -604,4 +604,12 @@ public void should_indicate_user_type_is_not_frozen() {
     PreparedStatement pst = session().prepare("SELECT v FROM not_frozen_table WHERE k = ?");
     assertThat(pst.getVariables().getType(0)).isNotFrozen();
   }
+
+  @Test(groups = "short")
+  public void should_handle_nested_udt_named_tuple() {
+    execute(
```

Comment:
For some reason google-java-format doesn't like these lines, running format collapses it until one line.